draft-js-plugins-editor/CHANGELOG.md

```diff
@@ -14,6 +14,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - `decorators` don't decorate plugins anymore.
 
+### Changed
+
+- *Breaking change* The `plugins` and `decorators` editor props need to be an Immutable.List()
```

Comment:
I have concerns regarding the API design here. All other props are plain functions, but only the plugins need the list. That's inconsistent and I'm worried about that.